### DISCUSSION TITLE
possible to use monoio?

### DISCUSSION THREAD

**Discussion by ouvaa** [tokio-rs/axum#2622] [Q&A]
possible to use monoio?
https://github.com/bytedance/monoio

**Comment by jplatte** [comment]
Yes, it should be possible.

I would start from the `serve-with-hyper` example, replace the `TokioIo` and `TokioExecutor` types with monoio-backed types (you will have to write these yourself unless somebody has already written a hyper/monoio integration library) and disable axum's default features in `Cargo.toml` to avoid having to compile tokio.
